src/share/util/shr_orb_mod.F90

```diff
@@ -78,8 +78,10 @@ real(SHR_KIND_R8) pure FUNCTION shr_orb_cosz(jday,lat,lon,declin,dt_avg)
       shr_orb_cosz =  shr_orb_avg_cosz(jday, lat, lon, declin, dt_avg)
    else
       shr_orb_cosz = sin(lat)*sin(declin) - &
-           cos(lat)*cos(declin) * &
-           cos((jday-floor(jday))*2.0_SHR_KIND_R8*pi + lon)
+      &              cos(lat)*cos(declin)*cos(jday*2.0_SHR_KIND_R8*pi + lon)
```

Comment:
@rljacob Your commit made this change on ACME:

    Fix greather-then-roundoff diff in cos zen calc
    
    For jday larger than one, the argument (2*pi*jday + lon) for cos() will be
    reduced by the cos function, and as a result, some accuracy will be
    lost.  The reduction is different for different machines and results in
    greather-then-roundoff differences between machines for the same case.
    Solution is to change argument to ((jday-floor(jday))*2*pi + lon)